src/config/network-filesystems.md

```diff
@@ -69,3 +70,39 @@ server status:
 
 You can use [nfs.conf(5)](https://man.voidlinux.org/nfs.conf.5) to configure
 your server.
+
+### Setting up a server (NFSv4-only, Kerberos disabled)
+
+If you don't plan on using NFSv3 even as a fallback option, you can set up an
+NFSv4-only server. This will let you eliminate the need for
+[enabling](./services/index.md#enabling-services) `statd` and `rpcbind`
+services, be it on server or client side.
+
+First, install `nfs-utils` and follow the instructions about `/etc/exports`
+listed above.
+
+Then, edit `/etc/nfs.conf` and disable version 3, as in:
+
+```
+[nfsd]
+vers3=n
+vers4=y
+vers4.1=y
+vers4.2=y
+```
+
+To verify which versions are enabled or disabled, run:
+
+```
+# cat /proc/fs/nfsd/versions
+```
+
+As the last step, set `NFSV4_ONLY` to a non-empty value in
+`/etc/sv/nfs-server/conf`
+
+```
+NFSV4_ONLY=true
+```
+
+Now you can [enable](./services/index.md#enabling-services) `nfs-server`
+service.
```

Comment:
This section should not exist. The last sentence of the preceding section should just say

> You can use [nfs.conf(5)](https://man.voidlinux.org/nfs.conf.5) to configure your server. In particular, to disable legacy protocol versions and support only NFSv4, add the following section:
>
> ```
> [nfsd]
> vers3=n
> vers4=y
> vers4.1=y
> vers4.2=y
> ```
>
> You can verify the configured list of supported versions by inspecting the contents of the file `/proc/fs/nfsd/versions`.

Any mention of the `/etc/sv/nfs-server/conf` file should be omitted until a final decision about how to best to check (or not) for the `statd` server in `nfs-server` has been rendered.